site/content/tutorial/02-reactivity/01-reactive-assignments/app-a/App.svelte

```diff
@@ -7,5 +7,5 @@
 </script>
 
 <button>
-	Clicked {count} {count === 1 ? 'time' : 'times'}
+	Clicked {count} {count === 0 || count === 1 ? 'time' : 'times'}
```

Comment:
This is wrong, it's "0 times" not "0 time"

Reply:
Reverted